docker-compose.wazo-router-confd.yaml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  wazo-router-confd:
+    image: wazopbx/wazo-router-confd:latest
+    ports:
+    - 8000:8000
```

Comment:
Please can you change the port to 8600 instead 8000. We don't want to use the default port from the debug server.  https://wazo-platform.org/contribute/new_service

Reply:
You can use port number 9600

Reply:
@sboily @mwolff44 8600 or 9600? how can I know which port is the next available one in the wazo-platform?

Reply:
sorry my mistake, it's 9600.

Reply:
We discuss to start at 9600 for c4.